### DISCUSSION TITLE
Global Configuration Nuxt3 Vue3

### DISCUSSION THREAD

**Discussion by JokerMartini** [Akryum/v-tooltip#1008] [Q&A]
Where do i define a global configuration file when using the Nuxt3 setup (module)? Does it go somewhere in the nuxt.config.js file? In my case i want to place the tooltips at the bottom. I wasn't really clear on how or where to do this based on the docs.

```js
export const config: FloatingVueConfig = {
  // Themes
  themes: {
    tooltip: {
      // Default tooltip placement relative to target element
      placement: 'bottom',
      // Delay (ms)
      delay: {
        show: 500,
        hide: 0,
      },
    },
  },
}
```
My nuxt.config.js
```js
// https://nuxt.com/docs/api/configuration/nuxt-config
export default defineNuxtConfig({
  devtools: { enabled: true },
  modules: ["@nuxtjs/tailwindcss", "@pinia/nuxt", "floating-vue/nuxt"],
  css: ["@/assets/css/tailwind.css"],
});
```

**Comment by JokerMartini** [comment]
I tried this and it didn't work...

```js
import FloatingVue from "floating-vue";

FloatingVue.options.themes.tooltip.placement = "bottom";

export default defineNuxtConfig({
  devtools: { enabled: true },
  modules: ["@nuxtjs/tailwindcss", "@pinia/nuxt", "floating-vue/nuxt"],
  css: ["@/assets/css/tailwind.css"],
});
```

**Comment by JokerMartini** [comment]
any get this working?

**Comment by Akryum** [comment]
Place this code in a [Nuxt plugin](https://nuxt.com/docs/guide/directory-structure/plugins) so it is run inside your application (not in Nuxt config file):

```js
import FloatingVue from "floating-vue";

FloatingVue.options.themes.tooltip.placement = "bottom";
```

**Comment by yoshrubin** [comment]
is there a migration guide for switching from nuxt2 to nuxt3
like where should I set the css?

**Reply by yoshrubin** [reply]
@Akryum I get an error if I just convert the plugin I have in Nuxt2 for Nuxt3 saying something about `compilerOptions` but then if I add the module as well I get a 500 error something about `length of undefined` in `VPopper mounted`
